charts/jaeger/templates/es-rollover-cronjob.yaml

```diff
@@ -67,6 +67,8 @@ spec:
             args:
               - rollover
               - {{ include "elasticsearch.client.url" . }}
+              {{ include "extra.cmdArgs" ( dict "cmdlineParams" .Values.esRollover.cmdlineParams ) | nindent 14  }}
+              {{- include "storage.cmdArgs" . | nindent 14 }}
```

Comment:
for my understanding, this change will allow us to pass in 
```
.Values.storage.elasticsearch.cmdlineParams
&
.Values.esRollover.cmdlineParams
```
both, Correct.

Reply:
yep exactly